### DISCUSSION TITLE
getToolbarEl() returning undefined

### DISCUSSION THREAD

**Discussion by esha-fatima** [GrapesJS/grapesjs#5282] [Q&A]
Upon using the default RTE with few options  (actions, styleprefix, adjustToolbar, and custom) passed in the config obj, getToolbarEl() returns undefined.

**Comment by artf** [comment]
Hi @esha-fatima are you able to provide a reproducible demo of your use case?

**Reply by esha-fatima** [reply]
Hi, so i was just wondering if it would be an issue if we used ckeditor plugin and set up the default RTE together in grap
<img width="589" alt="Screenshot 2023-08-10 at 11 23 17 AM" src="https://github.com/GrapesJS/grapesjs/assets/85702449/079fdf25-7cb9-43a3-8bd0-c3ef75e9dacc">
es js init()?

**Reply by esha-fatima** [reply]
@artf would these 2 co-existing conflict?

**Reply by artf** [reply]
no @esha-fatima if you're using a custom RTE, the default one is totally skipped

**Reply by esha-fatima** [reply]
Just a follow-up here that did this work in the releases prior to v21.x? I am guessing that the default RTE was at least created and it returned a valid toolbar regardless of a custom RTE (i.e ckeditor)

**Reply by artf** [reply]
The toolbar wrapper is actually still created, even with a custom RTE (you can check it in this [demo](https://grapesjs.com/demo-newsletter-editor.html)). 
Maybe the only difference is the creation was moved post plugins, which means you can have it once the editor is ready:
```sh
// your plugin
(editor) => {
  // ...
  editor.onReady(() => {
    editor.RichTextEditor.getToolbarEl()
  })
}
```
